base/timing.jl

```diff
@@ -103,7 +103,7 @@ Return the total amount (in bytes) allocated by the just-in-time compiler
 for e.g. native code and data.
 """
 function jit_total_bytes()
-    return Int(ccall(:jl_jit_total_bytes, Csize_t, ()))
+    return UInt(ccall(:jl_jit_total_bytes, Csize_t, ()))
```

Comment:
Is the conversion to `UInt` necessary since `Csize_t` is _already_ `UInt`: https://github.com/JuliaLang/julia/blob/3254e623468939d2e7b8bec50b6997e43f72246f/base/ctypes.jl#L59?